knox-homepage-backend/pom.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.apache.knox</groupId>
+        <artifactId>gateway</artifactId>
+        <version>1.4.0-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>knox-homepage-backend</artifactId>
```

Comment:
Why are these new modules prefixed with "knox" rather than "gateway"?

Reply:
I think it expresses more that they belong to Knox as a product, including the gateway, the CLI, the KnoxShell rather than referring only the gateway part.

Reply:
While some of the information supports the client pieces, I still feel that the information is all about the gateway. I could be persuaded otherwise, but this is my feeling at this point.

Reply:
The idea is to add `KnoxShell/Client Downloads and docs` later on into the `General Proxy Information` block, but it's been scoped out for now.
I gave a second thought about the module names and I think the UI should be kept as-is (`knox-homepage-ui`) because of the `General Proxy Information` block. However, the back-end module can - and should - be renamed like `gateway-service-metadata`.
What do you think?

Reply:
Sounds good to me. I was unaware of the client downloads bit.

Reply:
Renamed the module as discussed above.